config/default.yml

```diff
@@ -711,3 +711,10 @@ Rails/TimeZone:
 Rails/Validation:
   Include:
     - app/models/**/*.rb
+
+##################### Performance ##################################
+
+Performance/Detect:
+  # Defines what method to use instead of `select.first`, e.g.
+  # `detect` or `find`.
+  PreferredMethod: detect
```

Comment:
Isn't it better to use the configuration parameter `Style/CollectionMethods:PreferredMethods` to choose between  `detect` and `find`?